nautobot/dcim/choices.py

```diff
@@ -678,6 +678,7 @@ class InterfaceTypeChoices(ChoiceSet):
 
     # Virtual
     TYPE_VIRTUAL = "virtual"
+    TYPE_BRIDGE = "bridge"
```

Comment:
I see us creating these constants but never using them.

Is there an expectation that we will have some data validation/enforcement that an interface must be set to a bridge type before or during an assignment on another interface?

Reply:
IIRC from the NetBox implementation, an interface's `bridge` FK *may* point to a `TYPE_BRIDGE` interface, but can potentially point to other types as well (i.e. it doesn't *have* to be `TYPE_BRIDGE`). Would want to double-check this though.

Reply:
Yeah which has been added in this PR already to do the filtering in the GUI (like here: https://github.com/nautobot/nautobot/pull/1651/files/ef309e97eb2a9028f25aa1c8723872a9cd93ed30#diff-7f9497e709291d40988eaf60afaddbed036bd537dd1c4532d8fce6b20081968eR2630-R2633) but this would require an interface already being set to `TYPE_BRIDGE`. I like that restriction, and it should be better UX, but what I don't see is validation (either via a Serializer or better yet the `clean` method) to enforce this when doing things not over the API/CSV import.